apps/test/unit/templates/studioHomepages/CourseCardTest.js

```diff
@@ -1,90 +0,0 @@
-import React from 'react';
```

Comment:
Is the entire test being removed because there's nothing significant remaining to test?